lib/organization_gem_dependencies/cli.rb

```diff
@@ -3,18 +3,22 @@
 require 'base64'
 require 'io/console'
 require 'json'
-require 'optparse'
-
 require 'octokit'
+require 'optparse'
```

Comment:
While I don't think ruby style guide has any such recommendations, I follow the python import recommendations which are system packages first, then third party, then local separated by a newline.

With that, optparse should remain where it was, with oktokit and the last require line separated by a single newline.